README.md

```diff
@@ -34,18 +35,19 @@ class StudentInfo extends React.PureComponent {
          */
       }
     },
-    '/student/:studentId': {
+    'student/:studentId': {
```

Comment:
Is this a compatible change?

Reply:
Yes, it is just a schema without any impact.